index.html

```diff
@@ -29,5 +29,122 @@ <h4>CONNECT WITH US</h4>
             <li><a href="#"><i class="fab fa-medium-m"></i></a></li>  
         </ul>
     </section>
+
+    <div class="works-grid">
+        <section class="work-1">
+            <div class="work-img"><img src="./assets/images/Snapshoot-Portfolio.png" alt="Portfolio snapshoot"></div>
+            <div class="work-desc"><h1>Tonic</h1>
+            <ul class="frame">
+                <li class="canopy">CANOPY</li>
+                <li class="b-e-d">Back End Dev</li>
+                <li class="year">2022</li>
+            </ul>
+            <p  class="work1-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="framework">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+            </div>        
+        </section>
+
+        <section class="work-1">
+            <div class="work-img"><img src="./assets/images/Snapshoot-Portfolio1.png" alt="Portfolio snapshoot"></div>
+            <div class="work-desc"><h1>Multi-Post
+                Stories</h1>
+            <ul class="frame">
+                <li class="canopy">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2022</li>
+            </ul>
+            <p  class="work1-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="framework">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+            </div>        
+        </section>
+
+        <section class="work-1">
```

Comment:
- [ ] The same here you can use `div` tag instead of `section` what do you think :D